doc/ale-rust.txt

```diff
@@ -23,6 +23,11 @@ Integration Information
      over cargo. rls implements the Language Server Protocol for incremental
      compilation of Rust code, and can check Rust files while you type. `rls`
      requires Rust files to contained in Cargo projects.
+  3. analyzer -- If you have rust-analyzer installed, you might prefer using
```

Comment:
This messes up numbering of linters. This should be 4 and `rustfmt` should be 5.